src/test/com/Elessar/app/client/MyClientIntegTest.java

```diff
@@ -74,7 +75,14 @@ public User load(String userName) {
                         }
                 );
 
-        final MyServer server = new MyServer("localhost", serverPort, db, users, mode, serverMetricManager);
+        MsgSender msgSender = null;
```

Comment:
The test can make use spring as well.
But I guess it's ok to do it in a separate PR